hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBackoffMetrics.java

```diff
@@ -202,51 +204,68 @@ public void setMetricValue(AbfsBackoffMetricsEnum metric, long value) {
   1.RCTSI :- Request count that succeeded in x retries
   2.MMA :- Min Max Average (This refers to the backoff or sleep time between 2 requests)
   3.s :- seconds
-  4.BWT :- Number of Bandwidth throttled requests
-  5.IT :- Number of IOPS throttled requests
-  6.OT :- Number of Other throttled requests
-  7.NFR :- Number of requests which failed due to network errors
-  8.%RT :- Percentage of requests that are throttled
-  9.TRNR :- Total number of requests which succeeded without retrying
-  10.TRF :- Total number of requests which failed
-  11.TR :- Total number of requests which were made
-  12.MRC :- Max retry count across all requests
    */
-  @Override
-  public String toString() {
-    if (getMetricValue(TOTAL_NUMBER_OF_REQUESTS) == 0) {
-      return "";
+  private void getRetryMetrics(StringBuilder metricBuilder) {
+    for (RetryValue retryCount : RETRY_LIST) {
+      long totalRequests = getMetricValue(TOTAL_REQUESTS, retryCount);
+      metricBuilder.append("$RCTSI$_").append(retryCount.getValue())
```

Comment:
we can create a separate constants file for all these acronyms to make it easier to modify if needed later

Reply:
+1

All these constants are used specifically for Metric work. They qualify to be added into `MetricConstants` class. It make sesne to retain that class and add all these constants there itself.